inc/thorvg.h

```diff
@@ -18,7 +18,7 @@
 #include <string>
 
 #ifdef TVG_BUILD
-    #ifdef _MSC_VER
+    #if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Is it ok to leave `_MSC_VER` in `./src/lib/sw_engine/tvgSwMath.cpp` without modification?

Reply:
It looks like clang supports `__builtin_clz`, so I would recommend to also change `_MSC_VER` in that file to `#if defined(_MSC_VER) && !defined(__clang__)`

Reply:
done